pages/levels/1.vue

```diff
@@ -35,28 +38,49 @@ export default {
     },
 
     correct() {
-      return (
-        this.code.includes('alt="axolotl in an aquarium"') ||
-        this.code.includes("alt='axolotl in an aquarium'")
-      );
+      return this.code.match(/alt=["].*axolotl.*["]/) || this.code.match(/alt=['].*axolotl.*[']/);
     },
+
     screenReader() {
       return `Image: ${this.altValue || "axolotl.jpg"}`;
     },
+
+    hint() {
+      if (this.showHint) {
+      this.before= `<body>
+    <p>Axolotls are listed as critically endangered in the wild.</p>
+    <--! Add an \'alt\'-tag of “axolotl in an aquarium” to the image -->`;
+      } else {
+        this.before= `<body>
+    <p>Axolotls are listed as critically endangered in the wild.</p>`;
+    }
+      return this.showHint ? true : false;
```

Comment:
wozu haben wir dann die computed property überhaupt wenn sie einfach nur `this.showHint` zurückgibt?
außerdem ist das auch gleich `return this.showHint`